### PR TITLE
[andoiddebugbridge] Update README

### PR DESCRIPTION
**Pull request by clinique**
Added working compatibility mention with Freebox Pop Player.

### CODE REVIEW

**Comment by GiviMAD** [comment]
Morning! Thanks for your contribution.

Can you fix the DCO problems? Basically you need to undo that commit and force push another one with the correct sign off.

[Here is a link](https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330) with some info in case you don't know what it is about.

**Comment by clinique** [comment]
> Morning! Thanks for your contribution.
> 
> Can you fix the DCO problems? Basically you need to undo that commit and force push another one with the correct sign off.
> 
> [Here is a link](https://community.openhab.org/t/dco-check-signing-off-with-github-web-editor-explanation/83330) with some info in case you don't know what it is about.

Usually for small documentation changes this is not needed.

**Review by GiviMAD — APPROVED** [review]
Oh, I did know it.
Great improvement to the section.
LGTM!

File: bundles/org.openhab.binding.androiddebugbridge/README.md [hunk 1]
```diff
@@ -8,7 +8,15 @@ If you are not familiar with adb I suggest you to search "How to enable adb over
...
 ## Supported Things
 
-This binding was tested on the Fire TV Stick (android version 7.1.2, volume control not working) and Nexus5x (android version 8.1.0, everything works nice), please update this document if you tested it with other android versions to reflect the compatibility of the binding. 
+This binding was tested on :
+
+| Device             | Android version | Comments                   |
+|--------------------|-----------------|----------------------------|
+| Fire TV Stick      | 7.1.2           | Volume control not working |
+| Nexus5x            | 8.1.0           | Everything works nice      |
+| Freebox Pop Player | 9               |                            |
```
Comment (jlaur) [inline bundles/org.openhab.binding.androiddebugbridge/README.md:17]: A comment would be nice. Is it working or not working?

**Review by jlaur — APPROVED** [review]
Thanks!

**Pull request merged**